docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
done.
- added `image: metasploit`
- created binstubs for `msfvenom` and `msfconsole`
- updated `docker/README.md`
- updated pull request description